### ISSUE TITLE
Tag keys are not prefixed before invalidation

### ISSUE THREAD

**Issue by Swanty** [spiritix/lada-cache#27]
If you debug here:
https://github.com/spiritix/lada-cache/blob/bac563672c470b839099c4755ff34d470650a129/src/Spiritix/LadaCache/Invalidator.php#L54

you can see that tags are not prefixed with the `lada:` prefix

output from redis-cli monitor
```
1474295382.561930 [2 127.0.0.1:57456] "EXISTS" "lada:tags:database:receptes:table:taxonomy_group"
1474295382.567505 [2 127.0.0.1:57456] "SMEMBERS" "lada:tags:database:receptes:table:taxonomy_group"
1474295382.567958 [2 127.0.0.1:57456] "EXISTS" "lada:tags:database:receptes:table:taxonomy_group:row:6"
1474295382.568279 [2 127.0.0.1:57456] "SMEMBERS" "lada:tags:database:receptes:table:taxonomy_group:row:6"
1474295382.572728 [2 127.0.0.1:57456] "DEL" "lada:ca4f3478a25b1a65e6ba63b3d486464b"
1474295382.573023 [2 127.0.0.1:57456] "DEL" "lada:8878cfb82fadfd146ec585caa3b9df35"
1474295382.573387 [2 127.0.0.1:57456] "DEL" "lada:239982319ac7e7e3bde82a09befe8db7"
1474295382.573605 [2 127.0.0.1:57456] "DEL" "tags:database:receptes:table:taxonomy_group"
1474295382.573867 [2 127.0.0.1:57456] "DEL" "tags:database:receptes:table:taxonomy_group:row:6"
```

**Comment by spiritix** [comment]
Good find. We should also extend the InvalidatorTest to make sure the tags get deleted properly.

**Comment by spiritix** [comment]
Fixed in #29
